blockly-svelte/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "svelte-app",
```

Comment:
Change name to match the other samples as: 
"blockly-svelte-sample".
Since this is a sample, I would se the version to "0.0.0" as well.

Reply:
updated